chips/stm32f4xx/src/clocks/clocks.rs

```diff
@@ -102,6 +102,7 @@
 //!
 //! Then, configure its frequency and enable it
 //! ```rust,ignore
+//! clocks.set_sys_clock_source(stm32f429zi::rcc::SysClockSource::HSI);
```

Comment:
When I wrote the usage example, I assumed boot configuration, hence the source clock is HSI.

Reply:
I understand. You also wrote to execute the tests just before `kernel::process::load_processes`. In this stage, the `sys_clock` might be already configured the something else (HSE), so it is required to set the source clock again according to what you want to test.
What do you think?

Reply:
I agree with you. The best solution in my opinion is to make clocks function independent of context. This would require that each test have a prologue configuring the clocks in the expected way before running the given test. What do you think?